### ISSUE TITLE
Correct `pyproject.toml` build-backend specification

### ISSUE THREAD

**Issue by nikitos3000** [pavdmyt/yaspin#107]
Currently:

https://github.com/pavdmyt/yaspin/blob/2c78d24eb2d1b31273f696e8d118b2c6721f1068/pyproject.toml#L75-L77

According to https://python-poetry.org/docs/pyproject/#poetry-and-pep-517

> If your `pyproject.toml` file still references poetry directly as a build backend, you should update it to reference poetry_core instead.

Should be updated  to,

```toml
[build-system]
requires = ["poetry_core>=1.0.0"]
build-backend = "poetry.core.masonry.api"
```

This will significantly speed up the package installation process, since the entire `poetry` dependency cone is currently being installed during isolated build per PEP517.

I can submit a PR if no objections.

**Comment by pavdmyt** [comment]
Hi @nikitos3000 

Thanks for pointing this out! I'll happily accept the PR.

**Comment by nikitos3000** [comment]
Hi @pavdmyt thank you for approving the PR. How do you feel about releasing a `1.3.1` version?

**Comment by pavdmyt** [comment]
I'll include this PR as well as a bunch of others in `v1.4.0`. If everything goes well, I'll do it this weekend.

**Comment by pavdmyt** [comment]
Included in [yaspin v1.4.0](https://github.com/pavdmyt/yaspin/releases/tag/v1.4.0)
